po/zh_CN.po

```diff
@@ -7489,22 +7489,22 @@ msgstr "LED点阵中每一个灯珠的高度"
 
 #. parameter name
 #: boxes/generators/matrix.py
-msgid "pysical_led_y"
+msgid "physical_led_x"
 msgstr ""
 
-#. help for parameter pysical_led_y
+#. help for parameter physical_led_x
 #: boxes/generators/matrix.py
-msgid "Width of the LED matrix pcb in mm"
+msgid "Width (X) of the LED matrix PCB in mm"
 msgstr "LED点阵pcb的宽度(mm)"
 
 #. parameter name
 #: boxes/generators/matrix.py
-msgid "pysical_led_x"
+msgid "physical_led_y"
 msgstr ""
 
-#. help for parameter pysical_led_x
+#. help for parameter physical_led_y
 #: boxes/generators/matrix.py
-msgid "Height of the LED matrix pcb in mm"
+msgid "Height (Y) of the LED matrix PCB in mm"
 msgstr "LED点阵pcb的高度(mm)"
```

Comment:
I assume the existing translations are correct in zh-CN, but this corrects them in the source msgid so that the strings still match. Please update zh-CN as part of PR approval/deployment, as I do not speak this language.